ecosystem/sep-0024.md

```diff
@@ -987,6 +989,8 @@ There is a small set of changes when upgrading from SEP-6 to SEP-24.
 * Solar wallet: https://solarwallet.io
 
 ## Changelog
+
+* `v2.7.0`: Add `refund_memo` and `refund_memo_type` parameters to withdraw endpoint. ([#1321](https://github.com/stellar/stellar-protocol/pull/1321))
```

Comment:
I think this should be 2.8.0, right?

I believe you forget to add the 2.7.0 changelog in https://github.com/stellar/stellar-protocol/pull/1320

Reply:
I think I messed up by actually incrementing the minor version on SEP-24 instead of the patch version. So 2.7 makes sense for this change.